k8s/base/madara/madara.dockerfile

```
@@ -0,0 +1,53 @@
+    $SCARB_REPO/$SCARB_VERSION/scarb-$SCARB_VERSION-$PLATFORM.tar.gz && \
+    tar -xz -C /usr/src/ --strip-components=1 -f $SCARB_TARGET && \
+    mv /usr/src/bin/scarb /bin
+
+# Build the application in release mode
+RUN cargo build --release
+
+# Stage 2: Create the final runtime image
+FROM debian:bookworm-slim
+
+# Install runtime dependencies
+RUN apt-get -y update && \
```

Hadolint

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)